node/overseer/src/lib.rs

```diff
@@ -817,7 +821,7 @@ where
 	async fn block_imported(&mut self, block: BlockInfo) -> SubsystemResult<()> {
 		let mut update = ActiveLeavesUpdate::default();
 
-		if let Some(parent) = self.active_leaves.take(&(block.parent_hash, block.number - 1)) {
+		if let Some(parent) = block.number.checked_sub(1).and_then(|number| self.active_leaves.take(&(block.parent_hash, number))) {
```

Comment:
in practice we won't import a block with number 0, but it caused a panic in a test